app/util/searchUtils.js

```diff
@@ -869,6 +869,57 @@ export const executeSearch = (getStore, refPoint, data, callback) => {
   debouncedSearch(getStore, refPoint, data, callback);
 };
 
+export function executeWidgetSearchImmediate({ input }, callback) {
+  const config = getConfiguration(null);
```

Comment:
The widget is still using digitransit API For place searches as can be seen in the request url of network tab in below screenshot. 

![image](https://user-images.githubusercontent.com/3275280/88731243-b64f0c00-d13f-11ea-8376-1834d6014967.png)

It seems to be caused by default config being loaded here, because `null` is equivalent to `'default'` in the `getConfiguration()` function. The `executeSearchImmediate` function above [seems to get config as param](https://github.com/kyyticom/digitransit-ui/blob/0944cd42fa4b8b3a90d14828d0d7aaffe7fe539f/app/util/searchUtils.js#L688) which is setup in the React component in [some kind of context](https://github.com/kyyticom/digitransit-ui/blob/0944cd42fa4b8b3a90d14828d0d7aaffe7fe539f/app/component/DTSearchAutosuggest.js#L173). It looks like the `getConfiguration` function is intended to be used in `server/` components and not `app/` components.

Unfortunately my expertise with React stops here and I have no idea how we get the correct configuration. Can we set the correct config somehow in the actual server code that serves `widget.js` or some kind of variable in the widget embed HTML 🤔 Probably same way can be used as setting config for the actual reittiopas webpage..

Reply:
yes its using default config.  context can't be used in widget.
What configuration file should be used for this widget?  should it be "mhdev"(config.mhdev.js) configuration? if yes,it should be injected to the widget during build. So that, the widget.js component will be having correct configurations..